### PR TITLE
Remove third party connections, update DAP code, fix Gemfile

### PR DESCRIPTION
**Pull request by konklone**
This eliminates the pings to `cdnjs.cloudflare.com` to `ombegov.github.io`. It moves the DAP `<script>` to reference the centrally hosted version (whose security properties are [described here](https://www.digitalgov.gov/2015/08/14/secure-central-hosting-for-the-digital-analytics-program/)).

It also fixes the `Gemfile` to pull `redcarpet` from rubygems instead of GitHub, as it was causing `bundle install` to fail. It looks like this repo doesn't currently make use of syntax highlighting at all, making redcarpet superfluous, but I left it in to make the smallest change possible.

### CODE REVIEW

**Comment by konklone** [comment]
My change to the `baseurl` broke the links, because it looks like the baseurl is expected to use `/a130/` before everything? I was confused, because it looks like `https://a130.cio.gov/` and `https://a130.cio.gov/a130/` are the same site?

My suggestion is to drop the extra `a130` from the baseurl (this was necessary when it was at `ombegov.github.io`, but not anymore) and from each link in the site, before the site gets heavily linked-to. In the meantime, I've updated the PR to fix the baseurl to use `a130/`.

File: _layouts/default.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <link href="{{ site.baseurl }}/assets/css/google-fonts.css" rel="stylesheet" type="text/css">
   <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/styles.css?{{ site.time | date: "%Y%m%j%H%M%S" }}">
   <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/syntax.css">
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <script src="{{ site.baseurl}}/assets/js/jquery.min.js"></script>
```
Comment (benbalter) [inline _layouts/default.html]: You can also use `site.github.url`, which will default to `/` locally.

Comment (konklone) [inline _layouts/default.html]: True, but if the repo can avoid using GitHub Pages-specific options, it's better for portability.

**Comment by jlberryhill** [comment]
Thanks, @konklone. Appreciate the fix!

**Pull request merged**